luagraphqlparser/lib.c

```diff
@@ -650,12 +650,26 @@ visit_fragment_definition(const struct GraphQLAstFragmentDefinition *def, void *
 static void
```

Comment:
Typo in commit message header (ttempt -> attempt)

Reply:
Seems resolved.